python/paddle/incubate/autograd/primx.py

```diff
@@ -656,8 +656,10 @@ def expand_nested_list(xs):
                     outputs[op.output_names[i]] = op.output(op.output_names[i])
 
                 attrs = {}
-                for name in sorted(op.attr_names):
-                    attrs[name] = op.attr(name)
+                runtime_attrs = op._get_runtime_attrs()
```

Comment:
Add comments to explain why we need this.

Reply:
done